src/applications/mhv/secure-messaging/sass/message-details.scss

```diff
@@ -91,16 +91,24 @@
         height: 20px;
       }
       @media (max-width: $small-screen) {
-        padding-left: 10px;
-        padding-right: 10px;
+        margin-right: 0;
+        margin-left: 0;
+        display: inline-block;

```

Comment:
the buttons may use some margins in mobile view 
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/87077843/35d596ec-4559-4c37-9fb4-d1663e61ca98)

Reply:
In mobile views at that width, the flex box ignores changes to the left/right margin and leaves them essentially zero no matter what you do by my experiments.  I think this is normal.